components/dashboard/src/app/AppBlockingFlows.tsx

```diff
@@ -31,11 +29,6 @@ export const AppBlockingFlows: FC = ({ children }) => {
         return <></>;
     }
 
-    // If orgOnlyAttribution is enabled and the user hasn't been migrated, yet, we need to show the migration page
-    if (shouldSeeMigrationPage) {
```

Comment:
Could you elaborate on this change? I'm not familiar. I want to understand why it is ok to remove now.

Reply:
I think the Org migration path dried out by now.

Reply:
It doesn't make sense to update code unused.

Reply:
ok